src/ui/templates/index.njk

```diff
@@ -16,17 +16,17 @@
     <link rel="apple-touch-icon" href="/assets/images/govuk-apple-touch-icon.png">
 
     <link rel="stylesheet" href="/assets/css/styles.css" />
-    <!--[if IE 8]>
```

Comment:
Note this has been removed, as `IE8` browser is not longer supported by MS and therefore we should not be supporting legacy browsers.